### PR TITLE
Move Country Above Region in UserProfile.cs

### PR DESCRIPTION
**Pull request by thabaum**
Moved country above region to help resolve any issues from trying to set the region prior to setting the country might be concerned.

https://github.com/dnnsoftware/Dnn.Platform/issues/2140 I can reference this issue but I will not say it will fix it.  But it makes more sense to have it show up here in this file so I propose the change.

### CODE REVIEW

**Comment by thabaum** [comment]
alright no additional commits!

**Comment by thabaum** [comment]
I was not changing the order in the UI, just in the code to match the UI and make the code get the country list set first I was hoping to help cure any issues regarding loading the UI automatically.

**Comment by thabaum** [comment]
@valadas I thought you had changed this already for newer installations?

**Comment by valadas** [comment]
> I was not changing the order in the UI, just in the code to match the UI

Oh I am sorry, I misunderstood this PR then, reopening...

File: DNN Platform/Library/Entities/Users/Profile/UserProfile.cs [hunk 1]
```diff
@@ -61,9 +61,9 @@ public class UserProfile : IIndexable
         public const string USERPROFILE_Unit = "Unit";
         public const string USERPROFILE_Street = "Street";
         public const string USERPROFILE_City = "City";
-        public const string USERPROFILE_Region = "Region";
         public const string USERPROFILE_Country = "Country";
-        public const string USERPROFILE_PostalCode = "PostalCode";
+        public const string USERPROFILE_Region = "Region";
+	public const string USERPROFILE_PostalCode = "PostalCode";
```
Comment (valadas) [inline DNN Platform/Library/Entities/Users/Profile/UserProfile.cs]: Can you just adjust the spacing here, other than that, I am fine with this. For other reviewers this does not change behaviour just the order of declarations so it matches in code what we have in the UI for most situations.

**Comment by thabaum** [comment]
Does it look right now?  When I saved it in GitHub it was moving all over the place but I think it settled in the right place this last time.

**Comment by valadas** [comment]
> Does it look right now? When I saved it in GitHub it was moving all over the place but I think it settled in the right place this last time.

Hmm, looks like in 5dd0331 it got pushed more than necessary, then in 2fb8727 it got back too much left.

Not a huge deal and we are planning to add an editor config to avoid those inconsistencies anyway, just a remark.

**Comment by valadas** [comment]
Might also be just a github UI glitch who knows, I will checkout the actual file after 9.4.2 is out...

**Pull request merged**